contrib/devtools/check-translations.py

```diff
@@ -130,9 +130,23 @@ def check_all_translations():
                 # pick all numerusforms
                 if numerus:
                     translations = [i.text for i in translation_node.findall('numerusform')]
+                    if len(translations) == 0:
+                        print(f'Numerus message without numerusform translations, needs fix.')
+                        have_errors = True
+                    if not translation_node.text.isspace():
```

Comment:
also needs to check `translation_node.text is not None`, or this can error out when there are child elements but no text.